src/Avalonia.Base/Threading/DispatcherPriority.cs

```diff
@@ -22,21 +22,6 @@ private DispatcherPriority(int value)
         /// </summary>
         public static readonly DispatcherPriority MinValue = new(0);
 
-        /// <summary>
-        /// The job will be processed when the system is idle.
-        /// </summary>
-        [Obsolete("WPF compatibility")] public static readonly DispatcherPriority SystemIdle = MinValue;
```

Comment:
We need a way to mark properties that are here to lighten the burden of porting WPF code, but shouldn't be used for the new code

Reply:
My suggestion is just to revert this change and leave this as is. Currently i can't think of a better solution

Reply:
What about Obsolete + EditorBrowsable(Never)?

Reply:
> What about Obsolete + EditorBrowsable(Never)?

If we want our users to use these fields when porting from WPF then i think EditorBrowsable isn't suitable here.

Reply:
> My suggestion is just to revert this change and leave this as is

Let's do it @Takoooooo

Reply:
@maxkatz6 Done